CHANGELOG.md

```diff
@@ -1 +1,20 @@
 # Dynatrace\.Event\_Driven\_Ansible Release Notes
+
+**Topics**
+
+- <a href="#v1-2-0">v1\.2\.0</a>
+    - <a href="#release-summary">Release Summary</a>
+    - <a href="#minor-changes">Minor Changes</a>
+
+<a id="v1-2-0"></a>
+## v1\.2\.0
+
+<a id="release-summary"></a>
```

Comment:
Might be best to use more specific ids if we want to use the same structure for future versions. Just an optional change though

Reply:
That's true. But they are created automatically by running the `antsibull `command for creating the changelog.